pkg/cri/server/podsandbox/sandbox_stop.go

```diff
@@ -111,27 +117,23 @@ func (c *Controller) stopSandboxContainer(ctx context.Context, sandbox sandboxst
 		}()
 	}
 
-	// Kill the sandbox container.
+	// Kill the pod sandbox container.
 	if err = task.Kill(ctx, syscall.SIGKILL); err != nil && !errdefs.IsNotFound(err) {
-		return fmt.Errorf("failed to kill sandbox container: %w", err)
+		return fmt.Errorf("failed to kill pod sandbox container: %w", err)
 	}
 
-	return c.waitSandboxStop(ctx, sandbox)
+	return c.waitSandboxStop(ctx, podSandbox)
 }
 
 // waitSandboxStop waits for sandbox to be stopped until context is cancelled or
 // the context deadline is exceeded.
-func (c *Controller) waitSandboxStop(ctx context.Context, sandbox sandboxstore.Sandbox) error {
-	select {
-	case <-ctx.Done():
-		return fmt.Errorf("wait sandbox container %q: %w", sandbox.ID, ctx.Err())
-	case <-sandbox.Stopped():
-		return nil
-	}
+func (c *Controller) waitSandboxStop(ctx context.Context, sandbox *types.PodSandbox) error {
```

Comment:
We can remove this function and call `sandbox.Wait(ctx)` directly.

Reply:
done